.github/workflows/cryptolake.yaml

```diff
@@ -0,0 +1,31 @@
+name: Cryptolake
+on:
+  workflow_run:
+    workflows:
+      - "Nessie"
+    branches:
+      - cryptolake
+    types:
+      - completed
```

Comment:
Does `completed` means that the workflow ended successfully?

Reply:
Nope, we have to check later on to make sure that the workflow_run.conclusion == "success"